### DISCUSSION TITLE
Support for more device types

### DISCUSSION THREAD

**Discussion by Silther** [libre-tube/LibreTube#1287] [Ideas]
What do you think about support for more device types?
Tablets and foldables are also mobile devices, but as they are mainly used in landscape mode, I don't think the current support is perfect.
I have already made a few suggestions in #1350.

**Comment by akano12** [comment]
-Project is open source
+
-No one is getting paid 

= no point in putting such effort especially when devs work on this during free time

**Reply by Silther** [reply]
It is quite pointless to criticise me for something I said 3 months ago. Besides, you made your point pretty clear in #2206 and now that Byrno has agreed with it too, there's no need for every know-it-all to harp on old discussions etc. because they're just "trivia".
Besides, the main reason for this thread was that no empty discussion should be displayed here, because I deleted the topic for reasons but you can't delete a disscusion.
